src/app/landing/LandingPage.jsx

```diff
@@ -0,0 +1,258 @@
+                media="(min-aspect-ratio: 2/3)"
+                srcSet="img/landing_page/map_tablet.png"
+              />
+              <img
+                className="OnboardingImage"
+                src="img/landing_page/map_narrow.png"
+                alt="map showing food, clothing, and personal care markers"
+              />
+            </picture>
+          </div>
+
+          <div className="Section">
+            <div className="OnboardingSectionNumber">2</div>
+
+            <div className="OnboardingText">
+              <h2>Find providers that will actually serve <span className="Emphasis">you</span></h2>
+              <p className="text-lighter">
+                Filter or answer a few questions and we will show you locations that meet your specific needs, from age to gender to family situations.
+              </p>
+            </div>
+
+            <img className="OnboardingImage" src="img/landing_page/welcome.png" alt="" />
+          </div>
+
+          <div className="Section">
+            <div className="OnboardingSectionNumber">3</div>
+
+            <div className="OnboardingText">
+              <h2>You’re all set!</h2>
+              <p className="text-lighter">
+                See the service details, go get what you need. We don’t keep your personal data, so your privacy is protected.
+              </p>
+              <GetStartedButton onClick={this.getStarted} primary fluid />
+            </div>
+
+            <picture>
+              <img
+                className="OnboardingImage"
+                src="img/landing_page/step3.png"
+                alt=""
+              />
+            </picture>
+          </div>
+        </div>
+
+        <div className="Testimonials Section">
+          <div className="TestimonialsInner">
+            <h3 className="TestimonialsTitle">GoGettas are saying</h3>
+            <Testimonial
+              img="img/landing_page/testimonial_jeffrey.png"
+              name="Jeffrey"
+              text="I know this information is good, it’s from people like me"
+            />
+            <Testimonial
+              img="img/landing_page/testimonial_kenia.png"
+              name="Kenia"
+              text="Usually, if I need something, I ask. But this is faster, easier, better."
+            />
+            <Testimonial
+              img="img/landing_page/testimonial_gayle.png"
+              name="Gayle"
+              text="Boom! I go in and there’s all this information."
+            />
+          </div>
+          <GetStartedButton onClick={this.getStarted} fluid className="btnWhite" />
+        </div>
+        <div className="Partners Section">
+          <h3>Collaboration</h3>
+          <h2>Love for the help we’ve had to create GoGetta!</h2>
+          <div className="PartnerPictures">
+            <PartnerPicture
+              link="http://www.aliforneycenter.org/"
+              alt="Ali Forney logo"
+              src="img/partners/ali_forney_logo.png"
```

Comment:
If these were moved to the /partners dir, doesn't that mean the /about page is now broken?
Not that I expect anyone to use it right now, but I wouldn't want a completely broken page. Maybe just fix the links there too, if it's easy?

Reply:
Fixed